.claude/skills/beads-issue-tracking.md

```
@@ -0,0 +1,97 @@
+```bash
+bd init       # rebuilds beads.db from .beads/issues.jsonl
+bd status     # verify
+```
+
+Do **not** set `no-db: true` in `.beads/config.yaml` unless you want to permanently disable SQLite. The SQLite DB gives you search, filtering, and daemon-backed performance.
+
+## Why the Pre-Commit Hook Sometimes Warns
+
+The `bd` pre-commit hook flushes pending changes to JSONL before every commit/stash. If the daemon is down and the DB is missing, it prints:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)